recipes/magics-metview-batch/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "Magics" %}
+{% set version = "4.2.6" %}
+
+package:
+  name: magics-metview-batch
+  version: {{ version }}
+
+source:
+  url: https://confluence.ecmwf.int/download/attachments/3473464/{{ name }}-{{ version }}-Source.tar.gz
+  sha256: 9b34a375d9125ab6e8a715b970da2e479f96370bac6a5bb8a015a079ed9e027c
+  patches:
+    - 0001-Find-proj_6_0.lib-if-proj.lib-isn-t-found.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pkg-config
+    - boost
```

Comment:
Can you use `boost-cpp` here?  The `boost` package on conda-forge include boost-python

Reply:
I guess you are using python here

Reply:
@scopatz - in fact, I could use boost-cpp - thanks for the suggestion, it's done now.